src/components/AllocationsComponent.jsx

```diff
@@ -121,22 +145,26 @@ export class AllocationsComponent extends Component {
 }
 
 const mapStateToProps = ({ allocationsList }) => {
-  const { allAllocations, allocationsCount, isLoading } = allocationsList;
-
+  const { allAllocations, allocationsCount, isLoading, activePage } = allocationsList;
   return {
     allAllocations,
     allocationsCount,
-    isLoading
+    isLoading,
+    activePage
   };
 };
 
 AllocationsComponent.propTypes = {
   isLoading: PropTypes.bool.isRequired,
-  allAllocations: PropTypes.array.isRequired,
+  allAllocations: PropTypes.object.isRequired,
   allocationsCount: PropTypes.number,
-  loadAllocationsAction: PropTypes.func.isRequired
+  loadAllocationsAction: PropTypes.func.isRequired,
+  loading: PropTypes.func.isRequired,
+  activePage: PropTypes.number,
+  resetAllocations: PropTypes.func.isRequired,
+  setActivePage: PropTypes.func.isRequired
 };
 
 export default withRouter(connect(mapStateToProps, {
-  loadAllocationsAction
+  loadAllocationsAction, loading, fetchData, setActivePage, resetAllocations
```

Comment:
Each method should be on it's on line. E.g.
```js
export default withRouter(connect(mapStateToProps, {
  loadAllocationsAction,
  loading,
  fetchData,
  setActivePage,
  resetAllocations
})(AllocationsComponent))

Reply:
This comment https://github.com/AndelaOSP/art-dashboard/pull/170#discussion_r234975057 has not been addressed